app/controllers/admin/pdfs_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Admin::PdfsController < ApplicationController
+  def new
+  end
+
+  def create
+    pdf = Pdf.new(pdf_params)
+    if pdf.save
```

Comment:
I agree I think this is good for now, but I wonder if we will need to change this around once we decided how to do the CMS part.